web/src/js/__tests__/ducks/flowsSpec.js

```diff
@@ -157,7 +157,9 @@ describe('flows actions', () => {
         file     = new window.Blob(['foo'], { type: 'plain/text' })
```

Comment:
If we are expecting `anything` on line 162, can we just eliminate these two lines?